backend/domain/VoteHandler.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * A class used to handle any voting related events or calculations for a Room.
+ * This includes tallying votes, and figuring out who the most voted player is.
+ */
+class VoteHandler {
+    constructor() {
+        this.daytimeVoteMap = {};
+        this.mafiaVoteMap = {};
+        this.medicChosenPlayer = null;
```

Comment:
Are we sure we don't want to make this a list to hold all potential healed players? Or is it needless complexity trying to encompass for the future?

Making a map will tell us who healed though I think

Reply:
I think its probably better to have that implemented in future iterations, in the case where we may want multiple medics

Reply:
Cool :)